### PR TITLE
These includes all the changes from Ethan Main

### PR DESCRIPTION
**Pull request by Inventorer**
The app.yaml is the same one we've all been using.  The main.py is the file we covered on the call today with all the changes Ethan made as well as the changes I made and Eric helped us make on the call.  The sendPost.html is the file Ethan made with a change I made to stringify the JSON and also to include the dictionary changes Eric suggested on the call.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import urllib
+import webapp2
+from google.appengine.ext import ndb
+
+
+class measurePair(ndb.Model):
+    name = ndb.StringProperty()
+    value = ndb.FloatProperty()
+
+class userMorphology(ndb.Model):
```
Comment (EricHennigan) [inline main.py:15]: nit: all class names should be CapWords

File: main.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    emailID = ndb.StringProperty()
+    mPairs = ndb.StructuredProperty(measurePair, repeated=True)
```
Comment (EricHennigan) [inline main.py:17]: nit: better name is "measurements"

File: main.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    date = ndb.DateTimeProperty(auto_now_add=True)
```
Comment (EricHennigan) [inline main.py:18]: blank line to signal the end of a block, before the method that follows.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        uPair = [measurePair(**item) for item in uM]  # this steps through the dictionary items from the dictionary list (uM), then it unpacks the dictionary item into measured pair, but it works because the input is assumed to have the form [{"name": "waistCircumference", "value": 32},{"name": "neckCircumference", "value": 16}]
+        userMorph = userMorphology(parent = ndb.Key("Profile", j['emailID'] or "*notitle*"),
+            emailID = j['emailID'], mPairs = uPair)
+
+#        userMorph.mPairs = uPair    # hand uPair list of objects to the mPairs (repeated = True) structure container
+        print "Morpho:  ", userMorph.mPairs
+        userMorph.put()
+
+        thisProfile = Profile(parent=ndb.Key("Profile",
+                                           j['emailID'] or "*notitle*"),
```
Comment (EricHennigan) [inline main.py:48]: Add a TODO that 'emailID' is a required field. Then, in a later change, we can check and enforce it, sending back a 400 Bad Request code https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

**Review by EricHennigan — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (v-size) [reply main.py:15]: done

File: main.py (hunk 2 above)
Comment (v-size) [reply main.py:17]: I used Measurement (singular) since the class refers to an individual name/value pair.

File: main.py (hunk 3 above)
Comment (v-size) [reply main.py:18]: done

File: main.py (hunk 4 above)
Comment (v-size) [reply main.py:48]: Added a comment in the code.  Where else should we add the TODO?

**Comment by EricHennigan** [comment]
This code is obsoleted by #6

**Pull request closed without merging**